packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -16,6 +16,7 @@ import 'icon_theme.dart';
 import 'icon_theme_data.dart';
 import 'icons.dart';
 import 'material.dart';
+import 'popup_menu.dart';
```

Comment:
We haven't been importing libraries just to make dartdoc links work.  Instead, our plan is to teach dartdoc how to link to classes you haven't imported.